test/testdata/packager/import-autocorrect/a.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 # typed: strict
-# selective-apply-code-action: source.fixAll.sorbet
```

Comment:
It looks you went away from the "test via code actions" approach, but you only deleted one of the two `.rbedited` files.

Reply:
Great catch, thanks! Just deleted the last `. rbedited` file